src/models/userDao.js

```diff
@@ -0,0 +1,155 @@
+const { AppDataSource } = require('./dataSource');
+
+const findById = async (id) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      id = ?
+    `,
+    [id]
+  );
```

Comment:
```suggestion
const findUserById = async (id) => {
  const [user] = await AppDataSource.query(
    `
    SELECT
      *
    FROM
      users
    WHERE
      id = ?
    `,
    [id]
  );
```
- `*` 말고 필요한 컬럼만 선택해서 골라주세요 :) id만 있어도 되겠네요!
- Dao단에서 선언하는 함수의 이름은 동사 뒤에 어떤 Entity를 조작하는지 명확이 하시는 것이 좋습니다. 함수명 전반적으로 모두 변경해 주세요 :)

Reply:
네 알겠습니다!